handlers.go

```diff
@@ -140,11 +140,15 @@ func (s *Server) ServeAPI() error {
 	e.GET("/get/:cid", s.handleGetFullContentbyCid)
 	// e.HEAD("/get/:cid", s.handleGetContentByCid)
 
+	//move /user route to separate /keys route, keep auth level
+	keys := e.Group("/keys")
+	keys.Use(s.AuthRequired(util.PermLevelUser))
+	keys.GET("", withUser(s.handleUserGetApiKeys))
+	keys.POST("", withUser(s.handleUserCreateApiKey))
+	keys.DELETE("/:keyid", withUser(s.handleUserRevokeApiKey))
+
 	user := e.Group("/user")
 	user.Use(s.AuthRequired(util.PermLevelUser))
-	user.GET("/api-keys", withUser(s.handleUserGetApiKeys))
```

Comment:
These are breaking changes, do we intend to change the current v1 endpoints? I thought we're going to create a new set but will be grouped together on `api/v2` instead?

if so, instead of removing, we should add the following:

```
v2Group  := e.Group("api/v2") // this should be global V2 group for all new v2 endpoints.

userV2 := v2Group.Group("/user")
userV2.GET("", withUser(s.handleUserGetApiKeys))
userV2.POST("", withUser(s.handleUserCreateApiKey))
userV2.DELETE("/:keyid", withUser(s.handleUserRevokeApiKey))
```

We don't need to remove the previous one, just introduce a new group for V2 and call the same handler.

Reply:
+1 ^ i had this same confusion on https://github.com/application-research/estuary/pull/603#discussion_r1034042625.  can we wait for https://github.com/application-research/estuary/pull/596 to merge so we can make these changes to `dev` on the v2 API group instead of using the `api-v2` branch? cc @gmelodie @en0ma

Reply:
we'll rebase them after #596 is merged. For now let's leave it this way